installer/scripts/service_control

```diff
@@ -302,107 +494,143 @@ disable_omsagent_service()
     local omsagent_service=${systemd_dir}/$OMSAGENT_WS.service
     local omsagent_initd=/etc/init.d/$OMSAGENT_WS
     if [ -f $omsagent_service ]; then
-        echo "Unconfiguring OMS agent (systemd) service ..."
+        echo "INFO:  Unconfiguring OMS agent (systemd) service $WORKSPACE_ID ..."
         /bin/systemctl -q disable $OMSAGENT_WS
         rm -f $omsagent_service
         /bin/systemctl daemon-reload
+        WS_STATUS=$?
     elif [ -f $omsagent_initd ]; then
-        echo "Unconfiguring OMS agent service ..."
+        echo "INFO:  Unconfiguring OMS agent service $WORKSPACE_ID ..."
         if [ -f /usr/sbin/update-rc.d ]; then
             /usr/sbin/update-rc.d -f $OMSAGENT_WS remove
         elif [ -x /usr/lib/lsb/remove_initd ]; then
             /usr/lib/lsb/remove_initd $omsagent_initd
         elif [ -x /sbin/chkconfig ]; then
             /sbin/chkconfig --del $OMSAGENT_WS > /dev/null
         else
-            echo "Unrecognized Service Controller to unregister OMS Agent Service."
+            echo "FATAL:  Unrecognized Service Controller to unregister OMS Agent Service."
             exit 1
```

Comment:
Assign a specific error code to this.
This may be the only fatal case where we should immediately exit; if we find no way to start a service, than no workspace service will be able to be started.

Reply:
I agree.  Please let's talk about that.  Some of these I looked for an existing one, and did not find it.  There may be more than one area, and I really am preferring to start a library for them, otherwise I can picture gobs of multiply duplicated work here.